client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/OpenApiConfigValidatorTest.java

```diff
@@ -18,7 +18,11 @@ class OpenApiConfigValidatorTest {
             "quarkus.openapi-generator.codegen.additional-api-type-annotations",
             "quarkus.openapi-generator.codegen.spec.spec_yaml.enable-security-generation",
             "quarkus.openapi-generator.codegen.type-mappings.UUID=String",
-            "quarkus.openapi-generator.codegen.spec.spec_yaml.type-mappings.UUID=String"
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.type-mappings.UUID=String",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-apis=false",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-models=false",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-apis=true",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-models=true",
```

Comment:
These additions are no longer necessary, are they?

Reply:
@hbelmiro  I removed them but in the process messed my repo up pretty good so I had to create a new PR: https://github.com/quarkiverse/quarkus-openapi-generator/pull/897